services/web-server/src/resolvers/Notify.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  Query: {
+    listDenylistAddresses(parent, { connection, filter }, { loaders }) {
+      return loaders.notify.load({ connection, filter });
```

Comment:
@helfi92 I got the server running but the queries seem to be giving me an error. Say, running an example query:- 

```
# Write your query or mutation here
{
  listDenylistAddresses{
    edges
    {
      cursor
    }
  }
}
```

gives me the following response:-
```
{
  "errors": [
    {
      "message": "Cannot read property 'load' of undefined",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "listDenylistAddresses"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "stacktrace": [
            "TypeError: Cannot read property 'load' of undefined",
            "    at listDenylistAddresses (/home/ojaswin/TaskCluster/taskcluster-services/services/web-server/build/index.js:3185:29)",
            "    at field.resolve (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/apollo-server-core/node_modules/graphql-extensions/dist/index.js:128:26)",
            "    at resolveFieldValueOrError (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/graphql/execution/execute.js:479:18)",
            "    at resolveField (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/graphql/execution/execute.js:446:16)",
            "    at executeFields (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/graphql/execution/execute.js:293:18)",
            "    at executeOperation (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/graphql/execution/execute.js:237:122)",
            "    at executeImpl (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/graphql/execution/execute.js:85:14)",
            "    at Object.execute (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/graphql/execution/execute.js:62:35)",
            "    at /home/ojaswin/TaskCluster/taskcluster-services/node_modules/apollo-server-core/dist/requestPipeline.js:195:36",
            "    at Generator.next (<anonymous>)",
            "    at /home/ojaswin/TaskCluster/taskcluster-services/node_modules/apollo-server-core/dist/requestPipeline.js:7:71",
            "    at new Promise (<anonymous>)",
            "    at __awaiter (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/apollo-server-core/dist/requestPipeline.js:3:12)",
            "    at execute (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/apollo-server-core/dist/requestPipeline.js:179:20)",
            "    at Object.<anonymous> (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/apollo-server-core/dist/requestPipeline.js:131:35)",
            "    at Generator.next (<anonymous>)",
            "    at fulfilled (/home/ojaswin/TaskCluster/taskcluster-services/node_modules/apollo-server-core/dist/requestPipeline.js:4:58)",
            "    at process._tickCallback (internal/process/next_tick.js:68:7)"
          ]
        }
      }
    }
  ],
  "data": {
    "listDenylistAddresses": null
  },
  "extensions": {}
}
```

I believe that the error might be because of the following line due to `loaders.notify` being undefined but I'm not sure how to fix that. 

Can you please have a look :)